Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     ],
     targets: [
         .target(name: "ReactiveTask", dependencies: ["Result", "ReactiveSwift"], path: "Sources"),
-        .testTarget(name: "ReactiveTaskTests", dependencies: ["ReactiveTask", "Quick", "Nimble"]),
+        .testTarget(name: "ReactiveTaskTests", dependencies: ["ReactiveTask", "Quick", "Nimble"])
```

Comment:
Please revert these changes. These trailing commas should not have been removed.